corehq/apps/accounting/generator.py

```diff
@@ -36,7 +49,15 @@
 
 def instantiate_accounting_for_tests():
```

Comment:
The naming of this is already pretty clear... but to be safe, should we use the `@unit_testing_only` decorator here? Seems like this could cause some damage in prod.